Documentation/design-docs/code-versioning-profiler-breaking-changes.md

```diff
@@ -0,0 +1,29 @@
+# Code Versioning Profiler Breaking Changes #
```

Comment:
So you saying that code map is broken after tiered re jit? Adding method is fine I think